scripts/ci_check_no_file_changes.sh

```diff
@@ -44,11 +39,25 @@ do
     echo "Invalid check: \"${CHECK}\". Falling back to exiting with FAILURE code"
     exit 1
   fi
+  REGEXES=(${REGEXES[@]} ${REGEX})
+done
+echo
 
-  if [[ "${FILES}" =~ ${REGEX} ]]; then
-    echo "Detected changes... Exiting with FAILURE code"
-    exit 1
-  fi
+cat<<EOF
+CHANGED FILES:
+$FILES
+
+EOF
+
+for FILE in ${FILES}
+do
+  for REGEX in ${REGEXES[@]}
```

Comment:
nit:
```
In 1.sh line 54:
  for REGEX in ${REGEXES[@]}
               ^-----------^ SC2068: Double quote array expansions to avoid re-splitting elements.
```